icu4c/LICENSE

```diff
@@ -281,31 +281,29 @@ property of their respective owners.
 
 3. Lao Word Break Dictionary Data (laodict.txt)
 
- #  Copyright (c) 2013 International Business Machines Corporation
- #  and others. All Rights Reserved.
+ # Copyright (C) 2016 and later: Unicode, Inc. and others.
+ # License & terms of use: http://www.unicode.org/copyright.html
+ # Copyright (c) 2015 International Business Machines Corporation
+ # and others. All Rights Reserved.
  #
- # Project: https://github.com/veer66/lao-dictionary
- # Dictionary: https://github.com/veer66/lao-dictionary/blob/master/Lao-Dictionary.txt
- # License: https://github.com/veer66/lao-dictionary/blob/master/Lao-Dictionary-LICENSE.txt
- #              (copied below)
+ # Project: https://github.com/rober42539/lao-dictionary
```

Comment:
i assume this is the same dictionary?

Reply:
Yes, it has been at various places at various times, it was also previously at
http://lao-dictionary.googlecode.com/git/Lao-Dictionary.txt

Reply:
Yeah, I recenymovednit to GitHub, and am now trying to maintain it there. I'm working on a project to add technical vocabulary to it as well. Hopefully I will have some additional data once I can collect it and get it checked.